### PR TITLE
fix(vulnerability): replace python jose by pyjwt

### PR DESCRIPTION
**Pull request by jbarreau**
Python-jose (https://github.com/mpdavis/python-jose) who use ecdsa which have vulnerability with ecdsa dependency.
Now PyJWT (https://github.com/jpadilla/pyjwt) doesn't use ecdsa ; so it doesn't have this vulnerability.

### CODE REVIEW

File: src/_example/flask_sqlalchemy_package/demo/forest_admin/smart/order.py [hunk 1]
```diff
@@ -121,9 +121,7 @@ async def execute_export_json(context: ActionContext, result_builder: ResultBuil
 }
 
 
-async def refound_order_execute(
-    self, context: ActionContext, result_builder: ResultBuilder
-) -> Union[None, ActionResult]:
+async def refound_order_execute(context: ActionContext, result_builder: ResultBuilder) -> Union[None, ActionResult]:
```
Comment (jeffladiray) [inline src/_example/flask_sqlalchemy_package/demo/forest_admin/smart/order.py:124]: I guess this change is unrelated

Comment (jeffladiray) [reply src/_example/flask_sqlalchemy_package/demo/forest_admin/smart/order.py:124]: + refund instead of refound :p

**Review by jeffladiray — COMMENTED** [review]
Other than the small comment, lgtm

**Review by jeffladiray — APPROVED** [review]

File: src/_example/flask_sqlalchemy_package/demo/forest_admin/smart/order.py (hunk 1 above)
Comment (jbarreau) [reply src/_example/flask_sqlalchemy_package/demo/forest_admin/smart/order.py]: In another PR.
